package.json

```diff
@@ -210,6 +210,6 @@
     }
   },
   "lint-staged": {
-    "webapp/**/*.js": "eslint --cache --fix"
+    "**/*.{ts,tsx}": ["eslint --cache --fix --rule 'no-console:2' ", "prettier --write"]
```

Comment:
The no-console rule was not triggering, so I enforced it here as an `error`. Helps in development (warn), but not allowed in commits